resources/views/components/form/field.blade.php

```diff
@@ -1,9 +1,11 @@
 <div @if ($id ?? null) id="container-{{ $id }}" @endif
      class="space-y-1"
      @if ($disabled ?? false) data-disabled="true" @endif
-     @if ($readOnly ?? false) data-readonly="true" @endif>
+     @if ($readOnly ?? false) data-readonly="true" @endif
+     {{ $attributes }}
```

Comment:
`$attributes` will cause incompatibility with Laravel Livewire (for who use it). Please remove it.

Thanks

Reply:
I’m using live wire components and seems to work fine. I didn’t realize that the attributes bag can’t be passed in?

Reply:
I’ll look into adding more explicit properties then as we need the ability to add things like ‘autofocus’, ‘required’, ‘auto fill’ etc - non data bound attributes but still important

Reply:
`Field` component is inherited from all other form components. In the `Input` component, for example, `$attributes` property is already used within `<input .... />`: with your modify, we will find twice `$attributes` property in the input, and in any other form component.

Reply:
Ah whoops! That makes sense - will remove!

Reply:
I remember now why I added this, it was because on the password component, the `x-data` attribute wasnt being picked up and therefore alpinejs wasnt working for toggling that element (between type text and password). Once I added `$attributes`, it worked.

Reply:
Ok, I will see the password bug deeper.